kivy/uix/modalview.py

```diff
@@ -277,7 +277,8 @@ def on_touch_move(self, touch):
         return True
 
     def on_touch_up(self, touch):
-        if self.auto_dismiss and not self._touch_started_inside:
+        # Explicitly test for False as None occurs when shown by on_touch_down
+        if self.auto_dismiss and self._touch_started_inside is False:
             self.dismiss()
             return True
         super(ModalView, self).on_touch_up(touch)
```

Comment:
```suggestion
        super(ModalView, self).on_touch_up(touch)
        self._touch_started_inside = None
```

Reply:
Funnily enough, that occurred to me, but then I saw this comment in the code: that a modal view can only be opened once? This is does not seem to be the case, as this example proves?

https://github.com/kivy/kivy/blob/0df47fc686c66397931a97d839a929c0338b4fbf/kivy/uix/modalview.py#L214

Reply:
Ah! It has to be dismissed before it can be shown again :-) My bad...